packages/cf-wallet.js/src/app-instance.ts

```diff
@@ -17,11 +17,16 @@ export class AppInstance {
    */
   readonly id: AppInstanceID;
 
+  // Application-specific fields
   readonly appDefinition: Address;
   readonly abiEncodings: AppABIEncodings;
+  readonly timeout: BigNumber;
+
+  // Funding-related fields
   readonly myDeposit: BigNumber;
   readonly peerDeposit: BigNumber;
-  readonly timeout: BigNumber;
+  readonly beneficiaries?: string[];
```

Comment:
Why optional?

Reply:
If `ETH_TRANSFER` is the interpreter type, then `beneficiaries` is `[AddressZero, AddressZero]` which is meaningless data. If `TWO_PARTY_OUTCOME` then it has correct data.

Reply:
Hmm right. Just seems a bit awkward to be required to know interpreter-level behavior to specify parameters this high level.

Reply:
Yeah. I have another PR coming to make it more explicit.